crates/turbopack-dev/src/chunking_context.rs

```diff
@@ -332,9 +332,11 @@ impl ChunkingContext for DevChunkingContext {
     }
 
     #[turbo_tasks::function]
-    fn asset_path(&self, content_hash: &str, extension: &str) -> FileSystemPathVc {
-        self.asset_root_path
-            .join(&format!("{content_hash}.{extension}"))
+    fn asset_path(&self, content_hash: &str, basename: &str, extension: &str) -> FileSystemPathVc {
```

Comment:
Nit: could we have the basename come before the content_hash in the order of the arguments, to match what's actually getting written to disk?

```suggestion
    fn asset_path(&self, basename: &str, content_hash: &str, extension: &str) -> FileSystemPathVc {
```

Reply:
The format is an implementation detail of turbopack-dev.

Production would probably use a different format (and won't use `basename` at all).

I guess we should probably pass `original_ident: AssetIdentVc` instead and do all the extension & basename stuff in turbopack-dev.